Sources/Routes/Root.swift

```diff
@@ -6,46 +6,75 @@ public class Root: Route {
     public init(){}
 
     public func handleRoute(request: Request) -> RouteData {
-        var body: String
-        if let files = getFilesFromDirectory(directory: request.directory){
-            body = prepareBody(body: files, method: request.method)
+        if (isPublicDirectory(request: request)) {
```

Comment:
Why does the root route handler need to check this? Based on your current design, I was expecting a dedicated handler for file browsing.

Reply:
Based on our conversation yesterday about exposing certain aspects of your code, I figured, perhaps I should only expose things that are labeled as "public." I can pull this out all allow it to file browse.

Reply:
I agree that limiting exposure, i.e., white-listing, is good. But why does that translate to implementing this logic in the root route?

Reply:
My thinking: Since root has the ability to list directories, if a user were to choose a directory other than the public directory, `./cob_spec/public`, they would see files and directories that they could not directly interact with. I guess this is fine, but it seemed strange to have if there was no route implementation for the other files and directories. Thus I decided to limit the ability to see other directories and files.

On the other hand, I could dynamically determine all available directories and files and turn them into routes, which would allow a user to browse all directories and files.